ar_exercise/exercise.rb

```diff
@@ -31,37 +31,107 @@ class Apartment < ActiveRecord::Base
+
+# Tenant.all.each { |t| puts t.name + " " + t.id.to_s }
+
 # Iterate over each apartment, for each apartment, display it's address and rent price
+
+# Apartment.all.each {|a| puts a.address + " " + a.monthly_rent.to_s}
+
 # Iterate over each apartment, for each apartment, display it's address and all of it's tenants
 
+# Apartment.all.each do |a|
+#    puts "Apartment: #{a.address}, Tenants:"
+#     a.tenants.each {|t| puts t.name}
+# end
+
+
 ################################################
 # CREATING / UPDATING / DELETING
 ################################################
 
 # Hint, the following methods will help: `new`, `create`, `save`, `uddate`, `destroy`
 
 # Create 3 new apartments, and save them to the DB
+# address, monthly_rent, sqft, num_beds, num_baths
+Apartment.where('address=? OR address=? OR address=?', "36 Southern Slope Dr", "15 Wake Forrest Rd", "1 Test St.").destroy_all
+
+Apartment.create(address: '36 Southern Slope Dr', monthly_rent: 2500, sqft: 3000, num_beds: 4, num_baths: 3)
+Apartment.create(address: '15 Wake Forrest Rd', monthly_rent: 500, sqft: 200, num_beds: 1, num_baths: 1)
+Apartment.create(address: '1 Test St.', monthly_rent: 900, sqft: 1000, num_beds: 2, num_baths: 2)
+
+# Apartment.all.each do |a|
+#    puts "Apartment: #{a.address}"
+#  end
+
 # Create at least 9 new tenants and save them to the DB. (Make sure they belong to an apartment)
+
+# Tenant.where('name=? OR name=? OR name=? OR name=? OR name=? OR name=? OR name=? OR name=? OR name=?', "Admiral Adama","Colonel Tye", "Starbuck", "Apollo", "Helio", "Dr Gaius Baltar", "The Doc", "Chief", "President Roslin").destroy_all
+#
+# Tenant.create(name: "Admiral Adama", age: 62, gender: "Male", apartment_id: 21)
+# Tenant.create(name:"Colonel Tye", age:62,gender: "Male", apartment_id: 21)
+# Tenant.create(name:"Starbuck", age:35,gender: "Female", apartment_id: 21)
+# Tenant.create(name:"Apollo", age:36,gender: "Male", apartment_id: 22)
+# Tenant.create(name:"Helio", age:32,gender: "Male", apartment_id: 22)
+# Tenant.create(name:"Dr Gaius Baltar", age:45,gender: "Male", apartment_id: 22)
+# Tenant.create(name:"The Doc", age:67,gender: "Male", apartment_id: 23)
+# Tenant.create(name:"Chief", age:38,gender: "Male", apartment_id: 23)
+# Tenant.create(name:"President Roslin", age:63,gender: "Female", apartment_id: 23)  #Hardcoded the apartment_id's. Was there a better way?
+#
+# Tenant.all.each {|t| puts t.name}
+
 # Note: you'll use this little bit of code as a `seeds.rb` file later on.
 
 # Birthday!
 # It's Kristin Wisoky's birthday. Find her in the DB and change her age to be 1 year older
 # Note: She's in the seed data, so she should be in your DB
+KW = Tenant.find_by(name: "Kristin Wisoky")
+
+# KW.age = KW.age + 1
+# puts KW
+
 
 # Rennovation!
 # Find the apartment "62897 Verna Walk" and update it to have an additional bedroom
 # Make sure to save the results to your database
 
+verna = Apartment.find_by(address: "62897 Verna Walk")
+
+puts verna[:num_beds] 
```

Comment:
Since these are AR instances / objects, it's generally preferred to use the `.` syntax to get properties:

```
verna.num_beds
```